src/node_internals.h

```diff
@@ -92,8 +92,10 @@ void SignalExit(int signal, siginfo_t* info, void* ucontext);
 std::string GetProcessTitle(const char* default_title);
 std::string GetHumanReadableProcessName();
 
-void InitializeContextRuntime(v8::Local<v8::Context>);
-bool InitializePrimordials(v8::Local<v8::Context> context);
+// TODO(RaisinTen): return a v8::MaybeLocal<v8::Context>.
+void InitializeContextRuntime(v8::Local<v8::Context> context);
```

Comment:
Suggesting a `MaybeLocal` instead of a `Maybe` because of the way the function is used here:
https://github.com/nodejs/node/blob/42d31447d56418021d3b984a1ed2ada3dd49918a/src/node_contextify.cc#L213-L217
Essentially, we can just return the context from the function and put the call inside the `if`.

Reply:
I think `InitializeContextRuntime()` should also return a `Maybe<bool>`, since it doesn’t create a new Context object but rather just tries to modify an existing one

Reply:
@addaleax Agreed, done!